SU-SWS/stanford_basic
# D8CORE-1402: add {grid: true} to autoprefixer

# READY FOR REVIEW

# Summary
- This fixes the problems we're having with IE

# Review By (Date)
soon

# Urgency
- Fixes readability issues for IE

# Steps to Test

1. Check out this branch
1. Clear cache
1. Look at your site in IE (https://www.browserstack.com/)
1. Verify that the links in the footer are not overlapping so that you cannot read them.

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1402
- Other PRs: Previous try: https://github.com/SU-SWS/stanford_basic/pull/152
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
